### DISCUSSION TITLE
What does "Number of downloadable items" means?

### DISCUSSION THREAD

**Discussion by vicki-leekx** [TumblThreeApp/TumblThree#428] [Q&A (Question & Answer)]
For example i had a blog that has 688 downloadable items but Tumblthree downloads 373 pictures instead of 688 pictures, why?

**Comment by thomas694** [comment]
"Downloadable items" is unequal to pictures. "Downloadable" means everything like pictures, videos, audio, text, and so on. Which of them are downloaded depends on the blog settings.

**Reply by vicki-leekx** [reply]
Oh i get it now, thanks
